pom.xml

```diff
@@ -1,68 +1,163 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
-   <parent>
-      <groupId>org.infinispan</groupId>
-      <artifactId>infinispan-persistence-parent</artifactId>
-      <version>9.4.0.Final</version>
-   </parent>
+   <groupId>org.infinispan</groupId>
    <artifactId>infinispan-cachestore-cassandra</artifactId>
-   <packaging>jar</packaging>
+   <version>9.4.0-SNAPSHOT</version>
```

Comment:
Since these live outside of the normal repo shouldn't they rely on Final builds? I don't think someone would want to copy this and rely on SNAPSHOT when it doesn't exist in their maven repo.

Reply:
Don't confuse the version of the cassandra store (which should be snapshot) with the dependency on the infinispan bits (which is at final).

Reply:
Aha I see further down, good 👍